### ISSUE TITLE
[sub-issue] CRw: Anchor links have the wrong alternate text "vlink 1"

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#273]
![image](https://cloud.githubusercontent.com/assets/7510842/5203661/0d28690e-7589-11e4-865b-cb3884b60668.png)

==> The symbol "v" of an anchor link should be hidden from screen readers.
The explanation (anchor link) is not necessary. But the "v" is incomprehensible.

**Comment by tonifisler** [comment]
Main issue is #340

**Comment by tonifisler** [comment]
Is this still an issue? I can't reproduce it, it's a pseudo-class `::before` and should not be read at all by the screen reader.

We'll reopen the issue if need be.
